xbmc/pvr/recordings/PVRRecordingsPath.cpp

```diff
@@ -32,10 +33,12 @@ CPVRRecordingsPath::CPVRRecordingsPath(const std::string& strPath)
   std::string strVarPath(TrimSlashes(strPath));
   const std::vector<std::string> segments = URIUtils::SplitPath(strVarPath);
 
-  m_bValid = ((segments.size() >= 4) && // at least pvr://recordings/[tv|radio]/[active|deleted]
-              StringUtils::StartsWith(strVarPath, "pvr://") && (segments.at(1) == "recordings") &&
-              ((segments.at(2) == "tv") || (segments.at(2) == "radio")) &&
-              ((segments.at(3) == "active") || (segments.at(3) == "deleted")));
+  m_bValid =
+      ((segments.size() >= 4) && // at least pvr://recordings/[tv|radio]/[active|deleted]/[rec|vod]
+       StringUtils::StartsWith(strVarPath, "pvr://") && (segments.at(1) == "recordings") &&
+       ((segments.at(2) == "tv") || (segments.at(2) == "radio")) &&
+       ((segments.at(3) == "active") || (segments.at(3) == "deleted")) &&
+       ((segments.at(4) == "rec") || (segments.at(3) == "vod")));
```

Comment:
This line reads wrong. Both indexes should be 4 here, right?

Reply:
Fixed